### ISSUE TITLE
oneDNN version planned for Tensorflow v2.5

### ISSUE THREAD

**Issue by ratan** [oneapi-src/oneDNN#994]
Tensorflow v2.4.1 has oneDNN version 1.6.4 integrated. Any idea what oneDNN version will be included in Tensorflow branch v2.5, so that one can know what all new feature coming into corresponding oneDNN version?

**Comment by vpirogov** [comment]
@ratan,

We plant to upgrade oneDNN to v2.2 in Tensorflow v2.5. This is of course contingent on the PR being accepted by Tensorflow maintainers.

**Comment by alenik01** [comment]
@vpirogov, thanks for the heads-up, these are great news.

> This is of course contingent on the PR being accepted by TensorFlow maintainers.

By the way is there some features list or roadmap from TF side of what is planned to be included in v2.5 release?

**Comment by vpirogov** [comment]
@alenik01,

Let me bring @agramesh1, Tensorflow enabling lead, to address this question.

**Comment by alenik01** [comment]
Good morning, @vpirogov.
> We plant to upgrade oneDNN to v2.2 in Tensorflow v2.5

the [PR#47543 ](https://github.com/tensorflow/tensorflow/pull/47543) has been recently pushed with the upgrade to v2.1. May you please comment a bit on the plans/timeline for v2.2?

**Comment by agramesh1** [comment]
@alenik01 We are working on it, the plan is still to update to v2.2 before the branching of TF2.5.

**Comment by alenik01** [comment]
Good morning, @agramesh1. Many thanks for clarifying this. And for v2.2 there will be a separate PR, correct?
I am asking because we also preparing a PR to TF at the moment.

**Comment by vpirogov** [comment]
Tensorflow just switched to rls-v2.2 branch (https://github.com/tensorflow/tensorflow/pull/48035). We are going to update it to production oneDNN v2.2 release once it is out early next week.
